### ISSUE TITLE
Still unable to follow anyone on hachyderm.io

### ISSUE THREAD

**Issue by ambihelical** [hachyderm/community#448]
### Please describe the issue you are seeing.

I am not able to follow anyone on Hachyderm.io. My instance (qoto.org) is neither silenced nor blocked by hackyderm.io, although it has been in the past for reasons I do not understand, and I believe must have been a misunderstanding.  In any case it no longer is in either category, but I'm still not able to follow anyone on hachyderm.io. 

This issue was supposed to be resolved with a new upstream version of mastondon but as I see Hachyderm.io is on 4.1.4 and it is still not resolved, I am resubmitting this issue. 
This is essentially the same issue as https://github.com/hachyderm/community/issues/250, but that was closed before resolution by some user preskton for expressed reasons that don't make any sense to me.

In any case, the issue hasn't been resolved and I would appreciate it if this issue could be kept open until resolution so that I can be notified about any potential solution. 

### How long have you noticed this issue?

8 months, probably a few more

### What geographic region are you accessing Hachyderm from?

san diego, ca

### Is there anything else that we should know?

This sucks

**Comment by hachyderm-admin** [comment]
Hello! Thank you for opening a new issue about this. It looks like we're going to need to dig into it as we were not able to replicate with some test follows.

First thing we'd like to check: would you feel comfortable sharing your qoto username? As part of the initial issue we had to perform some cleanup tasks, and we're wondering if your account is specifically stuck for some reason. If you're not comfortable sharing here, please send us an email referencing this ticket to admin@hachyderm.io .

Can you provide some more information:

* Are you able to see posts from Hachyderm on your home instance? That is, when you're looking at the Federated feed or elsewhere, can you see posts? 
* Do you see any error messages when you try to follow a Hachyderm user?
* -- For the error messages, you may also need to open the Console. In most Chrome based browsers you can do this by right clicking anywhere on the screen and choosing Developer Tools -> Inspect and clicking on the Console tab. You will need to reload the page. Note that on most web pages there can be several warn / error messages here, feel free to put in a screencap rather than trying to copy/paste.
* Are you trying to follow a user by searching for them and doing a follow, or by navigating directly to their profile and clicking follow (which opens a dialog box prompting your username@instance pattern to follow from)? Regardless of which you are using, do you see the same behavior by doing the other?
* Are you using the web page or one of the mobile apps? Is there any difference in behavior there?

**Comment by ambihelical** [comment]
B1 -  yes. Also I see replies from hachyderm.io users on threads. 
B2 - the initial follow seems to work, but when come back to it it the button says "Cancel follow request", or it eventually does.  I don't see any changes in my following list. Also I use Firefox.  If the errors may be hidden, I can install chrome.  Let me know if this would be fruitful. 
B3 - no errors
B4 -  the first.  Even when I paste the uid@instance pattern, it leads to the same page (qoto hosted mirror I guess?) with the same results. 
B5 - usually follow from web.  Metatext on ios has same issue.

**Comment by hachyderm-admin** [comment]
We were able to dive into this. 

First things first: thank you both for you patience in this, it took some time to dig around and see what the situation was. It looks like the original issue is spot on - for some reason, qoto wasn't resolving as part of that prior issue with Mastodon.

For context: a while ago, the way that Mastodon applied a server level Limit or Suspend involving iterating over all the accounts on that instance and applying a user-level limit or suspend. When the instance level moderation was removed, it took time for a way to be developed to try and undo this action. Originally it could not be differentiated from a regular user level limit or suspend.

For ambihelical, we took a look at your account and don't see an additional cause for a suspend (e.g. a report or some other trail), so we have manually unsuspended your account.

For the rest of the stuck qoto users (it's not all of them): we're looking into why it looks like a sizeable chunk of accounts on qoto didn't resolve when we expected.

Thank you both for bringing this back to our attention!

**Comment by Preskton** [comment]
We recently upgraded to v4.2.0; it's possible something in the update helped this situation. Could any of the folks on the thread check and report back?

**Comment by slamp** [comment]
@ambihelical @TimTr Could you check if the issue is solved ?

**Comment by slamp** [comment]
Good. Closing this issue then
